scripts/install-lib

```diff
@@ -21,7 +21,7 @@ is-enabled() {
     # Note that we must invert the condition so that the exit code indicates
     # a failure when the unit exists and is disabled or when the unit does
     # not exist
-    ! systemctl list-unit-files "$1" | awk "/$1/{exit \$2 == \"enabled\"}"
+    ! systemctl --quiet is-enabled "$1"
```

Comment:
```suggestion
    ! systemctl --quiet is-enabled "$1" 2>/dev/null
```

The original command didn't print any error for missing service files.

Reply:
Yes, I was wondering about suppressing the error message, I'll add the redirect of stderr.

But the inversion is still necessary:

```console
koan@tux:~$ sudo systemctl disable ssh.service
Synchronizing state of ssh.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable ssh
Removed /etc/systemd/system/sshd.service.
Removed /etc/systemd/system/multi-user.target.wants/ssh.service.
koan@tux:~$ systemctl --quiet is-enabled ssh.service; echo $?
1
koan@tux:~$ sudo systemctl enable ssh.service
Synchronizing state of ssh.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install enable ssh
Created symlink /etc/systemd/system/sshd.service → /lib/systemd/system/ssh.service.
Created symlink /etc/systemd/system/multi-user.target.wants/ssh.service → /lib/systemd/system/ssh.service.
koan@tux:~$ systemctl --quiet is-enabled ssh.service; echo $?
0
```

From the `systemctl` man page:

           Checks whether any of the specified unit files are enabled (as with enable). Returns an
           exit code of 0 if at least one is enabled, non-zero otherwise.

Reply:
Sorry, you're right about the inversion, I forgot the result of the if :)